CHANGELOG.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Changelog
+---
+
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased] - yyyy-mm-dd
```

Comment:
I don't really like the `yyyy-mm-dd`, and it doesn't seem to be in Keepachangelog either, why'd you add it?

Reply:
It was in the template I copied from. I thought I'd leave to make clear the format, but I think we both agree that this is the superior format and we have no American developers here)